terra-jupyter-python/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.0.3 - 10/18/2019
+
+- Update `terra-jupyter-base` version to `0.0.4`
+`Image URL: us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:0.0.4`
```

Comment:
this should be `us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:0.0.3` right?